geocoder/index.cpp

```diff
@@ -83,11 +83,25 @@ void Index::AddStreet(DocId const & docId, Index::Doc const & doc)
 {
   CHECK_EQUAL(doc.m_type, Type::Street, ());
   size_t const t = static_cast<size_t>(doc.m_type);
+
+  auto isStreetSuffix = [] (std::string const & s) {
```

Comment:
The name is misleading: the function is testing for street synonyms, not street suffixes.

Reply:
fixed